src/main/java/ch/njol/skript/expressions/ExprFacing.java

```diff
@@ -42,57 +42,65 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Facing")
-@Description("The facing of an entity or block, i.e. exactly north, south, east, west, up or down (unlike <a href='#ExprDirection'>direction</a> which is the exact direction, e.g. '0.5 south and 0.7 east')")
-@Examples({"# makes a bridge",
-		"loop blocks from the block below the player in the horizontal facing of the player:",
-		"\tset loop-block to cobblestone"})
-@Since("1.4")
+@Description({
+	"The facing or opposite facing of an entity or block, i.e. exactly north, south, east, west, up or down",
+	"unlike <a href='/expressions.html#ExprDirection'>direction</a> which is the exact direction, e.g. '0.5 south and 0.7 east'",
+	"NOTE: Changing the opposite facing has the same result of changing the actual facing."
+})
+@Examples({
+	"# Makes a bridge",
+	"loop blocks from the block below the player in the horizontal facing of the player:",
+	"",
+	"# Get the block's face you're looking at",
+	"on right click:",
+		"\tsend \"You're looking at %opposite facing of player% side of %type of targeted block%.\""
+})
+@Since("1.4, INSERT VERSION (opposite)")
 public class ExprFacing extends SimplePropertyExpression<Object, Direction> {
 	
 	private static final boolean useBlockData = Skript.isRunningMinecraft(1, 13);
 	
 	static {
-		register(ExprFacing.class, Direction.class, "(1¦horizontal|) facing", "livingentities/blocks");
+		register(ExprFacing.class, Direction.class, "[:opposite] [:horizontal] facing", "livingentities/blocks");
 	}
 	
-	private boolean horizontal;
-	
+	private boolean horizontal, opposite;
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		horizontal = parseResult.mark == 1;
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		horizontal = parseResult.hasTag("horizontal");
+		opposite = parseResult.hasTag("opposite");
 		return super.init(exprs, matchedPattern, isDelayed, parseResult);
 	}
 	
-	@SuppressWarnings("deprecation")
 	@Override
 	@Nullable
-	public Direction convert(final Object o) {
+	@SuppressWarnings("deprecation")
+	public Direction convert(Object o) {
 		if (o instanceof Block) {
 			if (useBlockData) {
 				BlockData data = ((Block) o).getBlockData();
 				if (data instanceof org.bukkit.block.data.Directional) {
-					return new Direction(((org.bukkit.block.data.Directional) data).getFacing(), 1);
+					return new Direction(opposite ? ((org.bukkit.block.data.Directional) data).getFacing().getOppositeFace() : ((org.bukkit.block.data.Directional) data).getFacing(), 1);
```

Comment:
I think it'd be best to try and shorten this line. Something like:
```suggestion
					org.bukkit.block.data.Directional data = (org.bukkit.block.data.Directional) data;
					return new Direction(opposite ? directional.getFacing().getOppositeFace() : directional.getFacing(), 1);
```